packager/packager.cc

```diff
@@ -1889,26 +1908,65 @@ class ComputePackageSCCs {
     static void run(core::GlobalState &gs) {
         Timer timeit(gs.tracer(), "packager::computeSCCs");
         ComputePackageSCCs scc(gs);
+        auto &db = gs.packageDB();
+
+        vector<core::packages::MangledName> allPackages;
+        absl::Span<const core::packages::MangledName> preludePackages;
+        absl::Span<const core::packages::MangledName> normalPackages;
+
+        {
+            auto packages = db.packages();
+            allPackages.insert(allPackages.end(), packages.begin(), packages.end());
```

Comment:
I find this a little odd: this is essentially a deepCopy of the package database. If we had done this with the symbol table, it would feel out of place.

Reply:
It's only the package names -- `db.packages()` returns a span of the mangled names. This is more like when we partition the vector of file refs into package and non-package in pipeline.cc.